Install with pnpm fails

When I add plug-and-play as a dependency in my project with pnpm, it fails with a  `ERR_PNPM_PREPARE_PACKAGE`  error.
From what I understand, pnpm tries to install plug-and-play child dependencies using npm (it should use whichever package manager is specified in plug-and-play's package.json `packageManager` field, but as it is empty, it defaults to npm).

```console
npm error code ERESOLVE
│ npm error ERESOLVE unable to resolve dependency tree
│ npm error
│ npm error While resolving: plug-and-play@2.5.9
│ npm error Found: typedoc@0.26.11
│ npm error node_modules/typedoc
│ npm error   dev typedoc@"^0.26.5" from the root project
│ npm error
│ npm error Could not resolve dependency:
│ npm error peer typedoc@"0.27.x" from typedoc-plugin-markdown@4.4.2
│ npm error node_modules/typedoc-plugin-markdown
│ npm error   dev typedoc-plugin-markdown@"^4.2.3" from the root project
│ npm error
│ npm error Fix the upstream dependency conflict, or retry
│ npm error this command with --force or --legacy-peer-deps
│ npm error to accept an incorrect (and potentially broken) dependency resolution.

```

A workaround is to run `pnpm install` this way : 
`npm_config_force=true pnpm install`

--- a/package.json
+++ b/package.json
@@ -38,8 +38,8 @@
     "should": "~13.2.3",
     "standard-version": "^9.5.0",
     "tsx": "^4.16.5",
-    "typedoc": "^0.26.5",
-    "typedoc-plugin-markdown": "^4.2.3",
+    "typedoc": "^0.27.7",
+    "typedoc-plugin-markdown": "^4.4.2",
     "typescript": "^5.5.4"
   },
   "exports": {

